mcbackend/core.py

```diff
@@ -152,11 +152,12 @@ def constant_data(self) -> Dict[str, numpy.ndarray]:
     def observed_data(self) -> Dict[str, numpy.ndarray]:
         return {dv.name: ndarray_to_numpy(dv.value) for dv in self.meta.data if dv.is_observed}
 
-    def to_inferencedata(self, **kwargs) -> InferenceData:
+    def to_inferencedata(self, equalize_chain_lengths: bool=True, **kwargs) -> InferenceData:
```

Comment:
```suggestion
    def to_inferencedata(self, *, equalize_chain_lengths: bool=True, **kwargs) -> InferenceData:
```
let's make it an explicit kwarg